a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -535,10 +535,10 @@ def execute(self, loadbalancer, vthunder):
     @axapi_client_decorator
     def revert(self, loadbalancer, vthunder, *args, **kwargs):
         vlan_id = self.get_vlan_id(loadbalancer.vip.subnet_id, True)
-        if self.axapi_client.vlan.exists(vlan_id):
-            LOG.debug("Revert TagInterfaceForLB with VLAN id %s", vlan_id)
-            self.axapi_client.vlan.delete(vlan_id)
+        if self.is_vlan_deletable() and self.axapi_client.vlan.exists(vlan_id):
```

Comment:
Swap this around. If it doesn't exist it's faster to fail immediately. (Same with all checks below).

Reply:
Done.